lib/ProductOpener/GS1.pm

```diff
@@ -367,6 +367,10 @@ my %gs1_maps = (
 		"MON" => "month",
 		"DAY" => "day",
 	},
+
+	claimElementCode => {
```

Comment:
Maybe rename claimElementCode to claimElementCodeNoAdded so that we can have labels like "contains added sugars"